pkg/cli/admin/release/extract_tools.go

```diff
@@ -490,9 +493,10 @@ func (o *ExtractOptions) extractCommand(command string) error {
 		// copy the input to disk
 		if target.InjectReleaseImage {
 			var matched bool
-			matched, err = copyAndReplaceReleaseImage(w, r, 4*1024, exactReleaseImage)
+			releaseInfo := fmt.Sprintf("%s,%s", exactReleaseImage, releaseName)
```

Comment:
Hmm.. this change look like this is backwards incompatible...??

previously only `release-image-pullspec` was being set, now `release-image-pullspec,release-image-name` is being set.

So if I use this new oc to extract installer from an old release image, that installer binary will have invalid contents right???

Reply:
yes, i'll rework that